crates/egui_extras/src/loaders/image_loader.rs

```diff
@@ -57,6 +58,7 @@ impl ImageLoader for ImageCrateLoader {
         // 1. URI extension (only done for files)
         // 2. Mime from `BytesPoll::Ready`
         // 3. image::guess_format (used internally by image::load_from_memory)
+        let uri = decode_animated_image_uri(uri).map_or(uri, |(uri, _frame_index)| uri);
```

Comment:
```suggestion

        // TODO(lucasmerlin): Egui currently changes all URIs for webp and gif files to include
        // the frame index (#0), which breaks if the animated image loader is disabled.
        // We work around this by removing the frame index from the URI here
        let uri = decode_animated_image_uri(uri).map_or(uri, |(uri, _frame_index)| uri);
```

Lets add a todo here